lib/python/rose/bush.py

```diff
@@ -99,6 +101,10 @@ def cycles(self, user, suite, page=1, per_page=None, form=None):
             "host": self.host_name,
             "user": user,
             "suite": suite,
+            "cycles": cycles,
+            "is_option_on": is_option_on,
+            "tasks": tasks,
```

Comment:
Agree and removed, a different rose issue was previously causing these to be required and Ben fixed that yesterday so it now works without them.